tests/unit/components/textInput.test.js

```diff
@@ -0,0 +1,155 @@
+import {render, unmountComponentAtNode} from 'react-dom';
+import React from 'react';
+import {act} from 'react-dom/test-utils';
+import TextInput from '../../../compoments/textInput';
+
+let container = null;
+beforeEach(() => {
+  // setup a DOM element as a render target
+  container = document.createElement('div');
+  document.body.appendChild(container);
+});
+
+afterEach(() => {
+  // cleanup on exiting
+  unmountComponentAtNode(container);
+  container.remove();
+  container = null;
+});
+
+describe('textInput', () => {
+  test('Displays title', () => {
```

Comment:
Could we add spaces between the test blocks 🙏🏾 . A bit tricky to distinguish the start and end of a test.

Reply:
Added new lines after each test